src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -748,6 +889,23 @@ def _from_mt_rest_object(cls, mt_rest_obj):
             last_modified_date=mt_rest_obj.last_modified_date,
         )
 
+    def is_custom_strong_type(self):
+        return (
+            CustomStrongTypeConnectionConfigs.PROMPTFLOW_TYPE_KEY in self.configs
+            and self.configs[CustomStrongTypeConnectionConfigs.PROMPTFLOW_TYPE_KEY]
+        )
+
+    def convert_to_custom_strong_type(self):
```

Comment:
are we intent to make this public? add docstring for public apis

Reply:
add _ to make it private